examples/vision/python_yolov4/yolov4_inference.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Object Detection with YoloV4\n",
+    "This notebook is intended to be an example of how to use MIGraphX to perform object detection. The model used below is a pre-trained yolov4 from the ONNX model zoo. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os.path\n",
```

Comment:
`from os import path` 
I am seeing error otherwise by default:
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-2-1105bcf2cf4d> in <module>
      1 import os.path
      2 
----> 3 if not path.exists("./utilities/coco.names"):
      4     get_ipython().system('wget https://github.com/onnx/models/raw/master/vision/object_detection_segmentation/yolov4/dependencies/coco.names -P ./utilities/')
      5 if not path.exists("./utilities/yolov4_anchors.txt"):

NameError: name 'path' is not defined
```

Reply:
Fixed